### ISSUE TITLE
SOIL uses deprecated API for OpenGL 3.0+

### ISSUE THREAD

**Issue by Querijn** [eug/awesome-opengl#1]
SOIL on the public repo uses deprecated API, and requires a different version. @jpvanoosten updated it to use the correct api if you use 3.0+.

http://3dgep.com/wp-content/uploads/2014/02/SOIL.zip

**Comment by eug** [comment]
Thanks for point this out! I have downloaded the zip file and noticed that a few files are renamed or missing (compared to upstream version) - see lib directory. For that reason, I was not able to compile on Linux. Also, based on his [blog post](http://www.3dgep.com/texturing-and-lighting-with-opengl-and-glsl/), the author states _"I fixed this issue and provided a solution and project files for Visual Studio 2012 (vs11)..."_, it seems that the fix is only compilable on Visual Studio, therefore, is not that helpful. I suggest to create a repository fixing both issues (compilation and deprecated API) and then I will update the SOIL url/description. Otherwise, I will keep as it is.
